default/stringtables/en.txt

```diff
@@ -4277,14 +4277,18 @@ Robotic species are sapient machines. These species' max population can be incre
 PHOTOTROPHIC_SPECIES_TITLE
 Phototrophic Metabolism
 PHOTOTROPHIC_SPECIES_TEXT
-'''Phototrophic species live mainly or entirely on sunlight. These species' max population is greatly influenced by the brightness of the local star - the bigger the better. Growth specials do not benefit them.
+'''Phototrophic species live mainly or entirely on sunlight. These species' max population is greatly influenced by the brightness of the local star - the bigger the better - and the planet size. Growth specials do not benefit them.
```

Comment:
position of `the bigger the better` and `and the planet size` are a bit awkward.
Possible solutions:
... `brightness of the local star and the planet size.`
... `brightness of the local star and the planet size - the brighter or bigger, the better.`
... `brightness of the local star - the brighter the better.  The max population is also influenced by the planet size - the bigger the better.`